imagepruner/install.go

```diff
@@ -0,0 +1,26 @@
+package imagepruner
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	imageprunerv1 "github.com/openshift/api/imagepruner/v1"
+)
+
+const (
+	GroupName = "imagepruner.operator.openshift.io"
```

Comment:
@coreydaley do we need a new group for this? Can't this live as a top-level type within the image registry group?

Ex - type name is `ImagePruner`, viewed via `oc get imagepruner.imageregistry.operator.openshift.io/cluster`, etc...

Reply:
updated in next push